### PR TITLE
Updating AMIs to latest loader and clean CentOS 7 1901_01

### PR DESCRIPTION
**Pull request by bentsi**

### CODE REVIEW

File: tests/janus-graph.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ backends: !mux
             security_group_ids: 'sg-81703ae4'
             subnet_id: 'subnet-5207ee37'
             ami_id_db_scylla: 'AMI_ID'
-            ami_id_monitor: 'AMI_ID'
+            ami_id_monitor: 'ami-02eac2c0129f6376b' # Official CentOS Linux 7 x86_64 HVM EBS ENA 1901_01
             ami_id_loader: 'AMI_ID'
```
Comment (amoskong) [inline tests/janus-graph.yaml]: ami_id_loader isn't updated

File: tests/longevity-large-partition-4days.yaml [hunk 2]
```diff
@@ -65,7 +65,7 @@ backends: !mux
             subnet_id: 'subnet-ec4a72c4'
             ami_id_db_scylla: 'AMI-ID'
             ami_id_loader: 'AMI-ID'
```
Comment (amoskong) [inline tests/longevity-large-partition-4days.yaml]: ami_id_loader isn't updated

File: tests/longevity-large-partition-4days.yaml [hunk 3]
```diff
@@ -75,8 +75,8 @@ backends: !mux
                     security_group_ids: 'sg-059a7f66a947d4b5c'
                     subnet_id: 'subnet-088fddaf520e4c7a8'
                     ami_id_db_scylla: 'AMI_ID'
```
Comment (amoskong) [inline tests/longevity-large-partition-4days.yaml]: indent has problem of eu_west_1 block

File: tests/manager-regression-multiDC-set-distro.yaml [hunk 4]
```diff
@@ -31,7 +31,7 @@ backends: !mux
             subnet_id: 'subnet-ec4a72c4 subnet-5207ee37'
             ami_id_db_scylla: 'AMI_ID_EAST AMI_ID_WEST'
             ami_db_scylla_user: 'centos'
-            ami_id_loader: 'ami-050c06c3e20036edc' # Loader dedicated AMI
+            ami_id_loader: 'ami-059dd02510b5f5841' # Loader dedicated AMI
             ami_loader_user: 'centos'
```
Comment (amoskong) [inline tests/manager-regression-multiDC-set-distro.yaml]: ami_id_loader & ami_id_monitor of eu_west_1_and_us_west_2 are not updated

File: tests/master_doc.yaml [hunk 5]
```diff
@@ -158,7 +158,7 @@ backends: !mux
             subnet_id: 'subnet-10a04c75'
             ami_id_db_scylla: 'ami-19eca679'
             ami_db_scylla_user: 'centos'
-            ami_id_loader: 'ami-19eca679'
+            ami_id_loader: 'ami-059dd02510b5f5841' # Loader dedicated AMI
```
Comment (amoskong) [inline tests/master_doc.yaml]: ami_id_monitor of us_west_1 isn't updated

Comment (amoskong) [reply tests/master_doc.yaml]: ami_id_monitor & ami_id_loader of us_west_2 aren't updated.

**Comment by amoskong** [comment]
In us-east-1, there are two AMI (scylla-qa-loader-ami-latest & scylla-qa-loader-ami-v4), they were copied from same source instance (scylla-qa-monitor-ami-latest | i-0fb6159f150e3847d ).

I copied scylla-qa-loader-ami-latest in us-east-1 to other regions, the description of AMI in other regions is `[Copied ami-059dd02510b5f5841 from us-east-1] scylla-qa-loader-ami-latest`. I saw the copied AMI-ids are used in this PR.

Today I recopied scylla-qa-loader-ami-v4 in us-east-1 to other regions, the description of AMI in other regions is `[Copied ami-0878efb77c212f4d3 from us-east-1] scylla-qa-loader-ami-v4`, just want to make it clean in future.

There are also some remaining issue which was fixed in my https://github.com/scylladb/scylla-cluster-tests/pull/904/files
- update AMI id in tests/gemini_test.yaml
- updated ALL ami_id of loader & monitor tests/janus-graph.yaml
- fixed indent issue in of eu_west_1 block of longevity-large-partition-4days.yaml
- ami_id_loader & ami_id_monitor of eu_west_1_and_us_west_2 in manager-regression-multiDC-set-distro.yaml
- updated all ami_id of loader & monitor of all regions in master_doc.yaml
I had added comments in this PR about all above issue.

**Comment by bentsi** [comment]
@amoskong we need this in. If you find smth. that I missed please fire PR

**Comment by bentsi** [comment]
@amoskong can you please backport to 3.0, have to leave

**Comment by amoskong** [comment]
@bentsi This PR fixed almost all eu-west-1 & us-east-1 jobs which effected our important jobs.
We should merge and backport this PR first, then review my additional PR https://github.com/scylladb/scylla-cluster-tests/pull/922), the missing part isn't serious.

**Comment by amoskong** [comment]
I tried to backport AMI/loader patches from (v1.v2.v3.v4) one by one, but there are many repeated conflicts. We can just update it to latest one (monitor: v3, loader: v4)

**Comment by amoskong** [comment]
Sent a PR for the backport of 3.0:
- Backport PR #920 to SCT branch-3.0 #923

**Comment by amoskong** [comment]
> In us-east-1, there are two AMI (scylla-qa-loader-ami-latest & scylla-qa-loader-ami-v4), they were copied from same source instance (scylla-qa-monitor-ami-latest | i-0fb6159f150e3847d ).
> 
> I copied scylla-qa-loader-ami-latest in us-east-1 to other regions, the description of AMI in other regions is `[Copied ami-059dd02510b5f5841 from us-east-1] scylla-qa-loader-ami-latest`. I saw the copied AMI-ids are used in this PR.
> 
> Today I recopied scylla-qa-loader-ami-v4 in us-east-1 to other regions, the description of AMI in other regions is `[Copied ami-0878efb77c212f4d3 from us-east-1] scylla-qa-loader-ami-v4`, just want to make it clean in future.

Hi @bentsi @roydahan 

Let's treat the AMI-ids that are used in this PR as our formal v4. I had added `(used in sct yaml)` or `(not used in sct yaml)` for the `Name` of AMI (not `AMI Name`).

```
$ git grep ami_id|grep loader:|sed -e 's/.*ami_id_/ami_id_/g'|sort -n|uniq -c
   2 ami_id_loader: 'ami-0011d4855d80b3127' # Loader dedicated AMI
    \_ (us-west-2 AMI_NAME: scylla-qa-loader-ami-v4)
  37 ami_id_loader: 'ami-059dd02510b5f5841' # Loader dedicated AMI
    \_ (us-east-1 AMI_NAME: scylla-qa-loader-ami-latest)
  32 ami_id_loader: 'ami-07d0df7186ef5adc5' # Loader dedicated AMI
    \_ (eu-west-1 AMI_NAME: scylla-qa-loader-ami-latest)
   1 ami_id_loader: 'ami-0dad569a4bdbc86a1' # Loader dedicated AMI
    \_ (us-west-1 AMI_NAME: scylla-qa-loader-ami-v4)
```

image

**Pull request merged**